doc/security/specs/ownership_transfer/_index.md

```diff
@@ -12,18 +12,46 @@ ownership state of the device:
 *   _UNLOCKED_OWNERSHIP_: The device is ready to be assigned to a new owner.
 *   _LOCKED_OWNERSHIP_: The device is assigned to an owner and in operational mode.
 
-This document defines the ownership management functions that control the
-transitions between ownership states:
-
-### Unlock Ownership {#unlock-ownership-top}
-
-Implements transition from `LOCKED_OWNERSHIP` to `UNLOCKED_OWNERSHIP` state. The device must be in
-`UNLOCKED_OWNERSHIP` state before it can be assigned to a new owner.
-
-### Ownership Transfer (or Ownership Assignment) {#ownership-transfer-top}
-
-Implements transition from `UNLOCKED_OWNERSHIP` to `LOCKED_OWNERSHIP` stage. The rest of this document
-refers to this functionality as Ownership Transfer.
+The following steps describe the ownership transfer process in detail.
+
+1. The current owner of the device wipes any secret information they may have on it, and boots the device with an "unlock ownership" boot service request and a signature authenticating the request.
+At this point the device switches from a `LOCKED_OWNERSHIP` to an `UNLOCKED_OWNERSHIP` state, and the current owner will now be referred to as the "previous owner".
+    *  The device attestation process will specify that the device is in an unlocked state rather than attesting to any specific owner.
+    * The previous owner’s public keys are still stored after ownership is unlocked.
+Key derivations will also correspond to the unlocked state rather than the previous owner.
+1. The new owner creates a set of keys by any means they wish (doesn’t have to be on an OpenTitan device).
+These keys must include an RSA-3072 key pair for secure boot, an ECDSA-P256 key pair for unlocking ownership, and another ECDSA-P256 key pair for endorsing the next owner (see [Owner Keys]({{< relref "#owner-keys" >}})).
+    * The number of redundant keys stored is configurable up to around 2kiB of total key material.
+The new owner may choose, for example, to generate several RSA public/private key pairs.
+1. The new owner gives their public keys to the previous owner (which may be the manufacturer), who endorses them with a cryptographic signature.
+The Silicon Creator can also endorse new owners, although this will not happen often in practice.
+The endorser then creates an "ownership transfer payload", which includes the new owner’s public keys and their own ECDSA-P256 signature to authenticate the keys.
+1. The device should now be booted with a "transfer ownership" boot service request and with the ownership transfer payload to a special region of memory.
+During boot (specifically during the `ROM_EXT` stage), the device will check the signature on the payload against the endorser’s public key.
```

Comment:
"against the endorser’s public key, which is either the previous owner or silicon creator."

Reply:
Added some points below that elaborate on this point, ptal?